black.py

```diff
@@ -90,6 +90,10 @@ class CannotSplit(Exception):
     """
 
 
+class InvalidInput(Exception):
+    """Raised by :func:`lib2to3_parse` when its input fails all parse attempts."""
```

Comment:
You shouldn't say *who* raises the exception but what it signifies. Who raises the exception can change in the future.

Reply:
Fair enough, will change the one above this as well, then.